src/abstracts.ts

```diff
@@ -138,3 +138,9 @@ export interface BlockQueryProperties {
         shape: string;
     };
 }
+
+export interface ScratchVariable {
```

Comment:
I think I'd go more with `SB3ScratchVariableCollection` and `SB3VariableProperties`

Was thinking on the typedefs - the ones that come DIRECTLY from the sb3 file format should probaly be prefixed

Reply:
I see what you're saying! Implemented.